libsupport/include/katana/Cache.h

```diff
@@ -53,41 +53,33 @@ struct CacheStats {
   uint64_t insert_hit_count{0ULL};
 };
 
-template <typename Key, typename Value, typename CallerPointer = void*>
+template <typename Value>
 class KATANA_EXPORT Cache {
```

Comment:
Just to make sure, this makes a `Cache` an explicit holder of things that are backed by storage? That makes sense to me a good thing to be true, but wanted to make sure.

Reply:
Yes, and I added documentation to `PropertyCache `to make this clear.